packages/core/admin/admin/src/components/LanguageProvider/tests/index.test.js

```diff
@@ -54,9 +53,7 @@ describe('LanguageProvider', () => {
 
     expect(screen.getByText('English')).toBeInTheDocument();
 
-    act(() => {
-      userEvent.click(screen.getByText('CHANGE'));
-    });
+    fireEvent.click(screen.getByText('CHANGE'));
```

Comment:
Could you use `userEvent`?

Reply:
Let's keep one discussion about this topic open.